Movie_MVVM/Movie_MVVM/FilmInfo/FilmInfoTableViewController.swift

```diff
@@ -0,0 +1,158 @@
+// FilmInfoTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Вью с информацией о выбранном фильме
+final class FilmInfoTableViewController: UITableViewController {
+    // MARK: - Private properties
+
+    private let cellTypes: [CellTypes] = [.images, .overview, .actors]
+    private var trailers = [Trailer]()
+    private var actors: [Actor] = []
+    private var movie: Movie? {
+        willSet {
+            tableView.reloadData()
+        }
+    }
+
+    // MARK: - Public properties
+
+    var viewModel: FilmInfoViewModelProtocol?
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        viewModel?.fetchActors()
+        viewModel?.fetchTrailers()
+        updateActorsData()
+        updateTrailersData()
+    }
+
+    // MARK: - Public methods
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        viewModel?.viewDidDisapeardAction()
+    }
+
+    // MARK: - Private methods
+
+    private func configUI() {
+        navigationController?.navigationBar.largeTitleTextAttributes =
+            [
+                NSAttributedString.Key.foregroundColor: UIColor.white,
+                NSAttributedString.Key.font: UIFont.systemFont(ofSize: 30, weight: .bold)
+            ]
+        navigationController?.navigationBar.tintColor = .black
+        view.backgroundColor = UIColor(named: Constants.blueViewColorName)
+        configCell()
+    }
+
+    private func updateActorsData() {
```

Comment:
```suggestion
    private func bind() {
```
контролер не должен знать что там запрашиваться, объедений все кожуры в методе bind